docs/pages/en-us/Getting-Started.md

```diff
@@ -598,12 +388,18 @@ Following is an example for a SampleApp's `viewDocument` event, which takes a si
 
 The name of the event is included in the generated method name. In addition to the custom parameters (document id in this case), the caller must pass the `AuditChannel` object to be used, as well as the tenant id, user id, and correlation id.
 
-The method will throw an exception if the audit event could not be stored for some reason (for example, network failure).
+The method will throw an exception if the audit event could not be stored for some reason (for example, network failure or supplying unsupported tenantId characters).
```

Comment:
Are we now offering some form of validation on the tenantId?  If so great, if not we shouldn't suggest that we are in the document.